### ISSUE TITLE
[Feature] Make the website more interactive

### ISSUE THREAD

**Issue by MegaArthur** [openstreetmap/openstreetmap-website#2370]
I use OSM and contribute to it daily, but there is this one thing that's been bugging me for a while now: interactivity.

After succesfully adding data for lots of elements (or just if you're searching for something) you'd probably like to inspect that element on the osm.org website and see its' detail.

The current method to do that is:
- Rightclick near the feature and choose "query features"
- Wait for way too long for overpass to return too many results which you need to filter
- Look at the tags in the form of raw OSM tags, this isn't too user friendly (although mapper friendly)

Second method:
- Enable "OSM data view" which is already too many clicks
- Sort through an ugly blue overlay to find what you want
- Look at the tags in form (...)

I say there should be two additions to the OSM website:

1. A simple way to click on the rendered features from the get-go, like automatically and always displaying the OSM Data overlay transparently and at certain zoom levels only.

2. A nice UI for the properties of an object, like the current left pane, but with tags interpreted and shown nicely instead of as raw data. Removing (or moving to the bottom) the last changeset description would also help, it can be pretty confusing for new users.

Thank you

**Comment by 7RST1** [comment]
`A simple way to click on the rendered features from the get-go, like automatically and always displaying the OSM Data overlay transparently and at certain zoom levels only.`

That would be very nice!
I just wanted to add something regarding the quoted part. Maybe show just POIs instead of all the data at certain levels, to avoid lag?

Maybe an easier way to achieve this is to render a carto-map just without the POIs etched into it, and then openstreetmap.org could itself add the POIs as an interactive vector map?

**Comment by mmd-osm** [comment]
@MegaArthur : Point 1 you propose is already part of the Top 10 OSM tasks (see https://wiki.openstreetmap.org/wiki/Top_Ten_Tasks#Clickable_POIs_on_the_frontpage). It most likely requires us to switch to vector tiles first.

Point 2 sounds very much like the idea that has been described here: #706

**Comment by danieldegroot2** [comment]
@tomhughes This issue should be closed in favour of (and/or have title/description adjusted)
- [Top Ten Tasks: Clickable POIs on the frontpage](https://wiki.openstreetmap.org/wiki/Top_Ten_Tasks#Clickable_POIs_on_the_frontpage) ( see also https://github.com/osmlab/osm-planning/issues/16 ); and
- #706 ( see also https://github.com/osmlab/osm-planning/issues/21 )

**Comment by mmd-osm** [comment]
osm-planning was initiated by a few folks over at OSM US a few years back. I'm not sure if that's still a thing today, or more or less abandoned.

**Comment by bhousel** [comment]
Yeah I created it a long time ago, but osm-planning has been effectively dead for >3 years:  
https://github.com/osmlab/osm-planning/issues/25

It's weird to me that you'd frame it as an OSM-US thing.  We all work on the same OSM right?

**Comment by mmd-osm** [comment]
Sure, no bad intentions here. It seemed like the most active contributors were part of OSM US, though it was always open for everyone to bring up some cool new ideas.
